packages/components/pie-webc-core/src/test/mixins/formControlMixin/formControlMixin.browser.spec.ts

```diff
@@ -0,0 +1,99 @@
+import { test, expect } from '@sand4rt/experimental-ct-web';
+import { MockComponent } from './MockComponent.ts';
+
+test.describe('FormControlMixin', () => {
+    // IMPORTANT: Mounting and Unmounting the component before each test ensures that any tests that do not explicitly
+    // mount the component will still have it available in Playwright's cache (loaded and registered in the test browser)
+    test.beforeEach(async ({ mount }) => {
+        await (await mount(MockComponent)).unmount();
```

Comment:
Do you need the brace around the mount?  Should be enough here?`await mount(MockComponent).unmount()`

Reply:
yes, the brace is needed to access properties on the return type of `mount` other you get the error `Property 'unmount' does not exist on type 'Promise<MountResult<MockComponent>>'.ts(2339)`

Reply:
I see, I think in that case because it is returning the promise it is looking for the unmounted value you can destructure it and then await that. I think it is a little more readable that way.

```javascript
const { unmount } = await mount( MockComponent )
await unmount()
```

Reply:
I personally find that to be less readable, so if it's ok I will leave for now - destructuring the class method seems a bit strange to me

Reply:
I think if we are going to break it into two lines it should probably be:
```javascript
const component = await mount(MockComponent);
await component.unmount();
```

Reply:
Yeah, I am happy with Xanders approach if you both prefer

Reply:
LGTM 👍